docs/src/pages/start/vs-code-configuration.md

```diff
@@ -104,7 +104,7 @@ Then you need to tell VSCode to add a configuration to the debugger. The easiest
 
 In the example below, replace `package-name` with the `name` property from your `package.json` file:
 
-```jsonc
+```json
```

Comment:
@rstoenescu This is not valid json, since json does not allow comments. That's why the comment is highlighted in RED.